<launch>

  <!--Publishes our "/start" frame at a constant rate. -->
  <node name="startFrame" pkg="remote_nav" type="startFrame.py">
  </node>

  <arg name="rviz_config" default="$(find remote_nav)/rviz/pr2_map_img.rviz" />
  <!-- The length in meters of the one dimensional track the robit floats on. -->
  <arg name="track_length" default="1.0" />
  <!-- The frame by which we recognize the robot. For a turtlebot this is /base_footprint -->
  <arg name="robot_frame" default="/base_footprint" />
  <!-- The topic you publish to in order to send navigation goals. -->
  <arg name="nav_topic" default="/move_base_simple/goal" />
  <!-- The topic you publish to in order to cancel nav goals. -->
  <arg name="cancel_topic" default="/move_base/cancel" />

  <!-- the topic you publish to send head goals -->
  <arg name="head_server" default="/head_traj_controller/point_head_action" />

  <node name="remote_nav" pkg="remote_nav" type="myviz.py" >
  	<param name="robot_frame" value="$(arg robot_frame)"/>
  	<param name="rviz_config" value="$(arg rviz_config)"/>
  	<param name="track_length" value="$(arg track_length)"/>
    <param name="nav_topic" value="$(arg nav_topic)" />
    <param name="head_server" value="$(arg head_server)" />
    <param name="cancel_topic" value="$(arg cancel_topic)" />

  </node>
</launch>
